index.css

```diff
@@ -230,10 +230,65 @@ button:disabled {
   border-radius: 8px;
   padding: 0.5em;
   background-color: #f7f7f9;
-  margin-top: 0.5em;  
+  margin-top: 0.5em;
 }
 
 span {
   border: 0.5px solid #dfe1e6;
   margin: 0 12px;
 }
+
+/* ----------Contacts page styling begins here----- */
+
+.contact-me {
```

Comment:
- [ ] One of the objectives of this project is to understand how to parse a design template precisely. 
Kindly check your background image path, it's not displaying at the moment. 🤝️
The inputs are sticking to the right side, please add some margin or padding to solve the issue. ✍️

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/212930940-9e381975-5858-4806-adc0-8cd47c67c127.png" width="300">|<img src="https://user-images.githubusercontent.com/17243325/212930936-e7c420bf-0452-493b-850f-613f13a9905c.png" width="300">|